### PR TITLE
Add ignore_exit_status option to swiftlint action

### PR DESCRIPTION
**Pull request by mfurtak**
Addresses #1442

Option off (default):
screen shot 2016-04-07 at 8 23 03 pm

Option on:
screen shot 2016-04-07 at 8 22 38 pm

### CODE REVIEW

File: fastlane/lib/fastlane/actions/swiftlint.rb [hunk 1]
```diff
@@ -79,6 +89,25 @@ def self.authors
...
         [:ios, :mac].include?(platform)
       end
+
+      def self.handle_swiftlint_error(ignore_exit_status, exit_status)
+        failure_suffix = if ignore_exit_status
+                           'which would normally fail the build.'
+                         else
+                           'which represents a failure.'
+                         end
+        secondary_message = if ignore_exit_status
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/swiftlint.rb]: I'm not a big fan of this syntax since it's hard to read for non Ruby devs

Comment (mfurtak) [inline fastlane/lib/fastlane/actions/swiftlint.rb]: OK, I will change it

**Comment by mfurtak** [comment]
@KrauseFx let me know what you think.

**Comment by KrauseFx** [comment]
Looks great, maybe you want to add a short one-line note to `Actions.md`?

**Comment by mfurtak** [comment]
Yes, thanks for the reminder!

**Comment by KrauseFx** [comment]
Looks great, thanks for updating

**Pull request merged**